### ISSUE TITLE
Black header in dark mode even if "supportDarkMode: false"

### ISSUE THREAD

**Issue by Bardiamist** [onfido/onfido-ios-sdk#155]
### What was the expected behaviour?

White header

### What happened instead?

<details>
<summary>Black header</summary>

![Simulator Screen Shot - iPhone 11 - 2020-05-19 at 10 43 17](https://user-images.githubusercontent.com/22082342/82282517-34639980-99be-11ea-94ab-520cf1432aa8.png)

</details>

### Version info:

- Cocoapods (if applicable): 1.9.1
- Onfido SDK (Debug/Release): 18.1.0 (Debug)
- iOS: 13.4.1
- Xcode: 11.4.1 (11E503a)
- Device/Simulator: Simulator
- Device/Simulator language: English

### Integration configuration:

```
  ONAppearance *appearance = [[ONAppearance alloc]
                              initWithPrimaryColor:UIColor.redColor
                              primaryTitleColor:UIColor.greenColor
                              primaryBackgroundPressedColor:UIColor.yellowColor
                              secondaryBackgroundPressedColor:UIColor.purpleColor
                              fontRegular: @"SFUIText-Regular"
                              fontBold: @"SFUIText-Bold"
                              supportDarkMode: false];
  [configBuilder withAppearance:appearance];
```

### Steps to reproduce:

Enable "Dark Appearance" on iOS 13 and open Onfido.

**Comment by anuragajwani** [comment]
HI @Bardiamist 

I'll be looking into the issue now. Will try to reproduce this. However as from your previous issue https://github.com/onfido/onfido-ios-sdk/issues/154 we don't support `pageSheet` presentation style.

Anurag

**Comment by anuragajwani** [comment]
Hi @Bardiamist 

I am able to reproduce this problem. I'll create a ticket and update you once I have any progress on the issue.

Sorry for the inconvenience cause,
Anurag

**Comment by keremgund** [comment]
Hi @Bardiamist ,
Actually this problem is about having UINavigationController which supports dark mode and showing the UIViewController that OnfidoFlow returns inside of it. 
If you don't want to disable the dark mode only for Onfido SDK (keep enable for your app), then you can disable dark mode just for the UINavigationController that you're using to present SDK with
`overrideUserInterfaceStyle = .light`

In the mean time, we will be investigating issue to see if we can fix on our side.

Thanks,
Kerem

**Comment by Bardiamist** [comment]
I tried
```
if (@available(iOS 13.0, *)) {
  [rootViewController setOverrideUserInterfaceStyle:UIUserInterfaceStyleLight];
}
```
but seems that shold be in `viewDidLoad`.

So I disabled dark mode in full my React native app by
```
<key>UIUserInterfaceStyle</key>
<string>Light</string>
```
in `info.plist`.
